### PR TITLE
Long Class-Path may break the frontend #1571

### PR DESCRIPTION
**Pull request by Flaurite**
Fix https://github.com/jmix-framework/jmix/issues/1571

### CODE REVIEW

File: jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/JmixVaadinServletContextInitializer.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+import com.vaadin.flow.spring.VaadinServletContextInitializer;
+import org.springframework.context.ApplicationContext;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.ServletException;
+
+public class JmixVaadinServletContextInitializer extends VaadinServletContextInitializer {
```
Comment (gorbunkov) [inline jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/JmixVaadinServletContextInitializer.java:27]: Let's add javadoc that explains that this is a workaround for the issue, like you did in JmixTempJarManifestUtils class

File: jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/JmixVaadinServletContextInitializer.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            return;
+        }
+
+        super.onStartup(servletContext);
+    }
+
+    /**
+     * Listener sets provided class path to "java.class.path" property.
+     */
+    protected class SetupTempClassPathContextListener implements ServletContextListener {
```
Comment (gorbunkov) [inline jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/JmixVaadinServletContextInitializer.java:64]: Do we need two separate classes SetupTempClassPathContextListener and RestoreClassPathContextListener? It seems that we can have one class and create two instances of the same class passing different arguments to its constructor

File: jmix-flowui/flowui/src/main/resources/io/jmix/flowui/module.properties [hunk 3]
```diff
@@ -14,4 +14,6 @@
 # limitations under the License.
 #
 
-jmix.flowui.menu-config = io/jmix/flowui/menu.xml
\ No newline at end of file
+jmix.flowui.menu-config = io/jmix/flowui/menu.xml
+
+jmix.core.exclude-beans=contextInitializer
```
Comment (gorbunkov) [inline jmix-flowui/flowui/src/main/resources/io/jmix/flowui/module.properties:19]: Let's add a comment that explains why we add this

**Review by gorbunkov — APPROVED** [review]

File: jmix-flowui/flowui/src/main/resources/io/jmix/flowui/module.properties (hunk 3 above)
Comment (Flaurite) [reply jmix-flowui/flowui/src/main/resources/io/jmix/flowui/module.properties:23]: Fixed

**Pull request merged**